src/classes/HH_NameSpec_TEST.cls

```diff
@@ -95,18 +95,17 @@ private with sharing class HH_NameSpec_TEST {
     static testMethod void TwoContactSameLastname() {
         if (strTestOnly != '*' && strTestOnly != 'TwoContactSameLastname') return;
                 
-        list<Contact> listCon = new list<Contact>();
-        Contact c1 = new Contact (
-            Salutation = 'Skipper',
-            FirstName = 'John',
-            LastName = 'Doe'
-        );
-        listCon.add(c1);
-        Contact c2 = new Contact (
-            FirstName = 'Jane',
-            LastName = 'Doe'
-        );
-        listCon.add(c2);
+        Contact[] listCon = new Contact[] {
```

Comment:
Please see the reply above for the similar comment. The change was made to reduce number of lines in the test class especially since we do not reference Contact cX (where X=1,2,...) anywhere in the test method. Other test methods that reference individual Contacts were not updated, even though they could have accessed records as listCon[0...X-1] instead.